src/orders-update.js

```diff
@@ -64,6 +64,21 @@ export default class OrdersUpdate {
       })
   }
 
+  resolveState (items) {
```

Comment:
We should be consistent by adding comments to this method like every other method

Cc @Siilwyn